api/server.py

```diff
@@ -134,7 +136,7 @@ def setup_elasticsearch_indexes():
                         "type": "text",
                         "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
                     },
-                    "outputs": {"type": "object", "enabled": False},
+                    "outputs": {"type": "nested"},
```

Comment:
Actually, now that we use the features index, this doesn't need to be nested anymore.